tcf_website/urls.py

```diff
@@ -14,11 +14,11 @@
     path('course/<int:course_id>', views.course_view, name='course'),
     path('course/<int:course_id>/<int:instructor_id>',
          views.course_instructor, name='course_instructor'),
-    path(
-        'instructor/<int:instructor_id>',
-        views.instructor_view,
-        name='instructor'),
+    path('instructor/<int:instructor_id>',
+         views.instructor_view, name='instructor'),
     path('reviews/new', views.new_review, name='new_review'),
+    path('reviews/<int:pk>/delete',
```

Comment:
Does inheriting `DeleteView` require that we use `pk` instead of something like `review_id`? This does not match other Django views in this file.

Reply:
yeah, it does! I tried using `review_id`like the other urls but it gave an error